app/Pomona/Routing/DataSourceRouteActionResolver.cs

```diff
@@ -69,15 +71,12 @@ private IPomonaDataSource GetDataSource(IPomonaSession session)
             return pr =>
             {
                 var request = pr;
-                var uriResolver = request.Session.UriResolver;
```

Comment:
@asbjornu You should have chosen the line above during conflict resolution, that's why the test is failing. 😺